Header header

int32     controller_id
float32   sample_period
float32[] pattern_0
float32[] pattern_1
float32[] pattern_2
float32[] pattern_3
float32[] pattern_4
float32[] pattern_5
